src/main/java/racingcar/car/NextStepCars.java

```diff
@@ -18,24 +21,42 @@ public NextStepCars(String carList) {
         }
     }
 
-    public List<Car> getNextStepCars() {
-        return nextStepCars;
-    }
-
     public int carCount() {
         return nextStepCars.size();
     }
 
     public List<String> getWinner() {
-        int winnerPosition = 0;
-        for(Car car : this.nextStepCars) {
-            winnerPosition = Math.max(winnerPosition, car.getPosition());
-        }
-        int finalWinnerPosition = winnerPosition;
+        int finalWinnerPosition = getFinalWinnerPosition();
         List<String> collect = this.nextStepCars.stream()
-                .filter(car -> car.getPosition() == finalWinnerPosition)
+                .filter(car -> car.moveForward(0) == finalWinnerPosition)
                 .map(car -> car.getName())
                 .collect(Collectors.toList());
         return collect;
     }
+
+    private int getFinalWinnerPosition() {
+        int winnerPosition = 0;
+        for(Car car : this.nextStepCars) {
+            winnerPosition = Math.max(winnerPosition, car.moveForward(0));
+        }
+        int finalWinnerPosition = winnerPosition;
+        return finalWinnerPosition;
```

Comment:
finalWinnerPosition 지역변수는 제거해도 괜찮을 것 같아요 :)
```suggestion
        return winnerPosition;
```